google/cloud/spanner_v1/client.py

```diff
@@ -116,7 +116,7 @@ class Client(ClientWithProject):
 
     :type route_to_leader_enabled: boolean
     :param route_to_leader_enabled:
-        (Optional) Default True. Set route_to_leader_enabled as False to
+        (Optional) Default False. Set route_to_leader_enabled as False to
```

Comment:
`Set route_to_leader_enabled as True to Enable leader aware routing. Enabling leader aware routing would route all requests in RW/PDML transactions to the leader region.`